workflow/workflow-demos-test/src_test/ivy/IvyCmsTest.java

```diff
@@ -0,0 +1,33 @@
+package ivy;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Locale;
+
+import org.junit.jupiter.api.Test;
+
+import ch.ivyteam.ivy.environment.Ivy;
+import ch.ivyteam.ivy.environment.IvyTest;
+
+@IvyTest
+public class IvyCmsTest
```

Comment:
The same here. Write a test for ch.ivyteam.ivy.wfdemo.businessdata.DemoDataCreator which uses Ivy.repo()

Reply:
I tried to add a test around this DemoDataCreator, but had some issues with it...
First of all, is it possible that ES is not stopped correctly if a test is terminated?

And secondly I could not really add new data... Or I did something wrong...